src/client/todos/todo.styl

```diff
@@ -0,0 +1,26 @@
+.todo-item
+  font-size .875em
+  margin-bottom .25em
+  max-width 42em
+
+  &__label, &__button
```

Comment:
The only difference that comes to my mind is explicitly binding label and button to a .todo-item container which makes it impossible to reuse that button in a different component/element. But todo-item button/label are not a good example for the above case as they are not designed to be reusable (otherwise they could've inherited some common properties from a global `button` class)